### PR TITLE
build: v1 and semantic versioning

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Bump versions to v1.0.0 and commit to start adopting semantic versioning.

This PR also set the release dates in CHANGELOG that way we can effectively release v1 today.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-- utils `v0.0.23`
-- nns-proto `v0.0.9`
+- nns `v1.0.0`
+- sns `v1.0.0`
+- cmc `v1.0.0`
+- ledger `v1.0.0`
+- ckBTC `v1.0.0`
+- ic-management `v1.0.0`
+- utils `v1.0.0`
+- nns-proto `v1.0.0`
```
Comment (lmuntaner) [inline CHANGELOG.md:12]: Don't we want to wait until ledger-icp for it?

Maybe wait until the LedgerCanister is not in nns.

Comment (peterpeterparker) [reply CHANGELOG.md:12]: No on the contrary, that's exactly the idea!

Release v1 and then v2 === breaking changes when we introduce ledger-icp/ledger-icrc

**Review by lmuntaner — APPROVED** [review]
I like the idea!

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md:12]: Discussed offline.

**Pull request merged**